symengine/utilities/matchpycpp/autogen_tests/test_case001.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file was automatically generated: DO NOT EDIT.
```

Comment:
Can we add information here exactly how it was generated and which file should be modified instead?

Reply:
This has been added below.